### ISSUE TITLE
Traffic Sign detection for visually impaired people

### ISSUE THREAD

**Issue by kyra-09** [TAHIR0110/ThereForYou#110]
**Is your feature request related to a problem? Please describe.**
for visually impaired people the most common problem is to be able to cross or walk on roads by themselves . for thyem to be able to do all these things I want to implemet  detection dl model for  traffic signs  for road-safety.

**Describe the solution you'd like**
I'd like to implemet various different  (CNN / RNN) pre-trained models using tranfer learning for the dataset and analyse which one has a better accuracy for traffic sign recognition.
Tools I'll use :- Numpy , keras , matplotlib , scikit-learn etc.
Dataset :- https://www.kaggle.com/datasets/ahemateja19bec1025/traffic-sign-dataset-classification

**Describe alternatives you've considered**

**Additional context**

GSSOC - 2024 Contributor .
@TAHIR0110 kindly assign me this issue

**Comment by TAHIR0110** [comment]
@kyra-09 assigned!

**Comment by AnoushkaIshi** [comment]
Can I work on this ?
